demo-v1/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "HubSpot calling integration sdk test app",
   "private": true,
   "scripts": {
-    "build": "webpack",
-    "start": "npm i && webpack-dev-server --open"
+    "build": "webpack && cp -a dist/. ../build",
+    "start": "yarn install && webpack-dev-server --open"
```

Comment:
Any particular reason for switching from `npm` to `yarn` to install dependencies?

Reply:
I've been using yarn for the demo-v1 app to be consistent with our internal dependency management tool though npm works too.

Reply:
As discussed, let's keep it npm for the time being until we do the monorepo setup!